packages/gafl-webapp-service/src/processors/__tests__/recurring-pay-helper.spec.js

```diff
@@ -28,14 +28,19 @@ describe('recurringPayReminderDisplay', () => {
 
 describe('validForRecurringPayment', () => {
   it.each([
-    [true, '12M', true, true],
-    [false, '8D', true, true],
-    [false, '12M', false, true],
-    [false, '12M', true, false]
-  ])('should return %s as licence length is %s, licence for you is %s and SHOW_RECURRING_PAYMENTS is %s', (expected, length, licenceFor, recurring) => {
-    process.env.SHOW_RECURRING_PAYMENTS = recurring
-    const permission = getPermission({ licenceFor, length })
-    const result = validForRecurringPayment(permission)
-    expect(result).toEqual(expected)
-  })
+    [true, '12M', true, true, undefined],
+    [false, '8D', true, true, undefined],
+    [false, '12M', false, true, undefined],
+    [false, '12M', true, false, undefined],
+    [false, '12M', true, true, 'telesales']
```

Comment:
Tiny point, but you're not testing the journey being a different string, only `undefined`. Put in a test for that and I think we're there.